service/r2/service.go

```diff
@@ -359,13 +369,21 @@ func (s *service) fetchRollupRuleHistory(w http.ResponseWriter, r *http.Request)
 	return s.sendResponse(w, http.StatusOK, data)
 }
 
+func (s *service) updateRuleSet(w http.ResponseWriter, r *http.Request) error {
+	data, err := s.handleRoute(updateRuleSet, r, s.metrics.bulkUpdateRuleSet)
```

Comment:
Do you still want to refer to the handler as `bulk...`?

Reply:
Mark and I had a discussion earlier up in the thread, and parts of it were in person. But the idea was that bulk kind of made it sound like you could update multiple rulesets at once. This name seemed more clear.